src/_includes/graphql/cart-object-24.md

```diff
@@ -5,17 +5,21 @@ Attribute |  Data Type | Description
 `applied_gift_cards` | [[`AppliedGiftCard`]][AppliedGiftCard] | An array of `AppliedGiftCard` objects. An `AppliedGiftCard` object contains the `code` text attribute, which specifies the gift card code. `applied_gift_cards` is a Commerce-only attribute, defined in the GiftCardAccountGraphQl module
 `applied_reward_points`| [`RewardPointsAmount`][RewardPointsAmount] | The amount of reward points applied to the cart
 `applied_store_credit` | [`AppliedStoreCredit`][AppliedStoreCredit] | Contains store credit information applied to the cart. `applied_store_credit` is a Commerce-only attribute, defined in the CustomerBalanceGraphQl module
+`available_gift_wrappings` | [GiftWrapping] | The list of available gift wrapping options for the cart
```

Comment:
Hi @keharper. Since I've been partially working on the PR with this feature, spotted one minor thing. 

```suggestion
`available_gift_wrappings` | [GiftWrapping]! | The list of available gift wrapping options for the cart
```